src/classes/PMT_PaymentWizard_CTRL.cls

```diff
@@ -339,7 +338,7 @@ public with sharing class PMT_PaymentWizard_CTRL {
     * @description Creates an opportunity query with payment mapping and currency fields, filtered on id paramter.
     * @return string An opportunity query string.
     */
-    public string makeOppQuery() {
+    private String makeOppQuery() {
```

Comment:
Could we spell out "Opportunity" here instead of "Opp"?

Reply:
Updated!